src/_utils/useFocusTrap/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useRef } from 'react'
+import createFocusTrap from 'focus-trap'
+
+const ESCAPE_KEY = 'Escape'
```

Comment:
Don't we have a key mapping in the utils?

Reply:
Waiting for #659 to be merged first, I'll add it there. Conflicts incoming :D